chain-impl-mockchain/src/block.rs

```diff
@@ -3,6 +3,8 @@ use crate::certificate;
 use crate::key::{Hash, PrivateKey, PublicKey, Signature, Signed};
 use crate::transaction::*;
 use chain_core::property;
+use num_derive::FromPrimitive;
+use num_traits::FromPrimitive;
```

Comment:
Do you have to use both of these traits for this to work?

Reply:
Yeah, I think they're both required.